### PR TITLE
Remove unneeded markup extensions

### LINKED ISSUE #115: Perspex.Markup.Xaml.MarkupExtensions.TypeExtension
A few issues:
- Couldn't this be in OmniXaml itself as it is framework-independent?
- It has a `[ContentProperty("TargetType")]` attribute but no `TargetType` property. Is that a mistake?

### LINKED ISSUE #1082: Remove our own TypeExtension ,StaticExtension etc
Now that we're on Portable.Xaml we can remove the `TypeExtension` and `StaticExtension` classes that we had to provide ourselves because OmniXaml didn't provide `{x:Type}` or `{x:Static}`.

### PR DESCRIPTION
**Pull request by jkoritzinsky**
Remove our Static and Type markup extensions since we can use the portable ones now.

Fixes #1082 

Fixes #115

### CODE REVIEW

**Comment by kekekeks** [comment]
We should start keeping track of breaking changes between versions somewhere.

File: samples/RenderTest/SideBar.xaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<Styles xmlns="https://github.com/avaloniaui">
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">>
```
Comment (grokys) [inline samples/RenderTest/SideBar.xaml]: There's an extra `>` here that's causing this XAML to fail on load.

**Review by grokys — CHANGES_REQUESTED** [review]
Looks good other than a misplaced '>' breaking RenderTest.

File: samples/RenderTest/SideBar.xaml (hunk 1 above)
Comment (jkoritzinsky) [reply samples/RenderTest/SideBar.xaml]: Fixed!

**Review by grokys — APPROVED** [review]

**Pull request merged**